app/src/main/java/mozilla/org/webmaker/WebmakerApplication.java

```diff
@@ -32,7 +32,8 @@ public void onCreate() {
         Router.sharedRouter().map("/main/:tab", MainActivity.class);
         Router.sharedRouter().map("/map/:projectId", Map.class);
         Router.sharedRouter().map("/projects/:projectId", Project.class);
-        Router.sharedRouter().map("/projects/:projectId/elements/:elementId", Editor.class);
+        Router.sharedRouter().map("/edit/:projectId/:elementId/:type", Editor.class);
```

Comment:
while i agree these routes need some work, probably doing in another ticket -- would you mind holding off? (will reduce merge conflicts too)